internal/sgbouncewizard/sgbouncewizard.go

```diff
@@ -313,40 +311,48 @@ func (srv *Server) ChangelogContext(next http.Handler) http.Handler {
 
 		limit, ok := queryParams["limit"]
 
-		if len(limit) > 1 {
-			paramError := errors.New("Duplicate Parameters")
+		if !ok || len(limit) > 1{
+			paramError := errors.New("Invalid limit Parameter: does not exist or to many")
+			http.Error(w, paramError.Error(), http.StatusBadRequest)
+			return
+		}
+
+
+		offset, ok := queryParams["offset"]
+		if !ok ||  len(offset) > 1 {
```

Comment:
What if they don't want to specify a limit or offset? For example, what if they want all the change_logs?

Reply:
Limit and offset were mainly done for pagination purposes because currently they are getting the entire DB and storing in the React state. That Is why I made it required.